docs/onboarding/rpc/chain.md

```diff
@@ -187,13 +189,9 @@ Gets a block from the chain from a hash or sequence
 }
 `} />
 
-import Terminal from '../../../src/theme/components/Terminal/Terminal'
-import GetBlock from '../../../src/theme/components/Terminal/rpc/GetBlock'
-
 ```
 curl -d '{"sequence": 1}' -X POST http://localhost:8021/chain/getBlock
 ```
```

Comment:
Do you want to keep the `curl` command without the terminal output?

Reply:
Yea I think it's still useful. Some discord users were having trouble with the `'` and `"` in the curl command. The Terminal animation is just so slow and the Terminal is too big so that's why I'm removing it.